tests/playwright/playwright.config.ts

```diff
@@ -42,15 +42,36 @@ export default defineConfig({
       name: 'webkit',
       use: { ...devices['Desktop Safari'] },
     },
-
-    /* Test against branded browsers. */
     {
       name: 'Microsoft Edge',
       use: { ...devices['Desktop Edge'], channel: 'msedge' },
     },
-    // {
-    //   name: 'Google Chrome',
-    //   use: { ..devices['Desktop Chrome'], channel: 'chrome' },
-    // },
+    {
+      name: 'cross-browser',
```

Comment:
Should we make a project per test type (mock-ui, deployment,...) and then specify the testDir in each?

Reply:
I think we need to do this per browser (webkit-ui, chromium-ui, webkit-accessibility) with the test dir specified and then have the triggering project (mock-ui) match 0 tests. Without this, the other projects default to running all tests and the triggering project also runs all tests (but using a default browser)

Reply:
Though this could get messy quite quickly!